Index: files/scorefiler_pessimism.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/files/scorefiler_pessimism.txt b/files/scorefiler_pessimism.txt
--- a/files/scorefiler_pessimism.txt	
+++ b/files/scorefiler_pessimism.txt	
@@ -1,0 +1,1 @@
+1316 128
Index: files/scorefiler_corner.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/files/scorefiler_corner.txt b/files/scorefiler_corner.txt
--- a/files/scorefiler_corner.txt	
+++ b/files/scorefiler_corner.txt	
@@ -1,0 +1,5 @@
+708 64
+ 34364 256
+ 33148 256
+ 33260 256
+2728 256
Index: files/scorefiler_manuell.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>2240 256\r\n1440 128\r\n2784 256\r\n1736 128\r\n2616 128\r\n1496 128\r\n3732 256\r\n3140 256\r\n2612 256\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/files/scorefiler_manuell.txt b/files/scorefiler_manuell.txt
--- a/files/scorefiler_manuell.txt	
+++ b/files/scorefiler_manuell.txt	
@@ -1,3 +1,7 @@
+5072 512
+ 31080 128
+ 3676 64
+1468 128
 2240 256
 1440 128
 2784 256
